src/main/java/com/snowflake/kafka/connector/internal/streaming/StreamingClientHandler.java

```diff
@@ -50,34 +52,61 @@ public static boolean isClientValid(SnowflakeStreamingIngestClient client) {
     return client != null && !client.isClosed() && client.getName() != null;
   }
 
+  /**
+   * Gets the Properties from the input connector config
+   *
+   * @param connectorConfig configuration properties for a connector
+   * @return the Properties object needed for client creation
+   */
+  public static Properties getClientProperties(Map<String, String> connectorConfig) {
+    Properties streamingClientProps = new Properties();
+
+    if (connectorConfig != null) {
+      streamingClientProps.putAll(
+          StreamingUtils.convertConfigForStreamingClient(new HashMap<>(connectorConfig)));
+    }
+
+    return streamingClientProps;
+  }
+
+  public static String getLoggablePropertyStr(Properties properties) {
+
+    return properties.entrySet().stream()
+        .filter(
+            propKvp ->
+                !StreamingUtils.SENSITIVE_STREAMING_CONFIG_PROPERTIES.contains(
+                    propKvp.getKey().toString()))
```

Comment:
Do we have the issue about lowercase VS uppercase?

Reply:
We don't have a case concern because it is looking at the key which should be the same constant string value. However i will tolowercase all of it just to be safe